src/main/java/com/google/sps/classes/SqlConstants.java

```diff
@@ -49,10 +29,6 @@
   public static final int USER_INSERT_MAJOR = 5;
   public static final int USER_INSERT_ISMENTOR = 6;
 
-  // Constants used for inserting mentor experience.
-  public static final int MENTOR_EXPERIENCE_INSERT_ID = 1;
-  public static final int MENTOR_EXPERIENCE_INSERT_TAG = 2;
-
   // Constants to update mentor evidence.
   public static final int MENTOR_EVIDENCE_PARAGRAPH = 1;
   public static final int MENTOR_EVIDENCE_USERID = 2;
```

Comment:
Are these constants still being used anywhere? If not, feel free to clean this up also!

Reply:
Oh, wow, they are not. Deleted!